src/webview/components/Molecules/WaterLevelPawPrint/WaterLevelPawPrint.tsx

```diff
@@ -8,7 +8,7 @@ interface WaterLevelPawPrintProps {
   value3: number;
 }
 
-const WaterLevelPawPrint: React.FC<WaterLevelPawPrintProps> = ({ value1, value2, value3 }) => {
```

Comment:
Recommend not using React.FC anymore I was looking into it and its fully redundant in modern React now 😭